manifest.json

```diff
@@ -100,6 +100,18 @@
                     "type": "string",
                     "description": "Path to the file describing your custom runtime configuration.",
                     "default": "runtime.json"
+                },
+                {
+                    "name": "useFeederCan",
```

Comment:
It will break compatibility by changing default behavior. 
Suggestion: use "true" for useFeederCan as default and false for mock service as default or
Release changes with MAJOR version update.

Reply:
That was the plan anyways. Keeping the old behavior and forcing an additional step on the users is not good user experience IMHO.